test/test_client.py

```diff
@@ -614,12 +614,10 @@ def test_init_disconnected(self):
 
         c = rs_or_single_client(connect=False)
         self.assertEqual(c.codec_options, CodecOptions())
-        self.assertIsInstance(c.max_bson_size, int)
```

Comment:
We should keep these tests. `c.max_bson_size` et al. should still work correctly even if it's deprecated.

Reply:
Reverted